### PR TITLE
Package satysfi-class-jlreq.0.0.2

### PR DESCRIPTION
**Pull request by abenori**
### `satysfi-class-jlreq.0.0.2`
A document class for SATySFi

---
* Homepage: https://github.com/abenori/satysfi-class-jlreq/
* Source repo: git+https://github.com/abenori/satysfi-class-jlreq.git
* Bug tracker: https://github.com/abenori/satysfi-class-jlreq/issues

---
:camel: Pull-request generated by opam-publish v2.0.3
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [x] Add to snapshot `snapshot-develop` :: satysfi-class-jlreq.0.0.2
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-5`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-stable-0-0-6` :: satysfi-class-jlreq.0.0.2

### CODE REVIEW

File: packages/satysfi-class-jlreq/satysfi-class-jlreq.0.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "satysfi-base" {>= "0.0.2"}
+  "satysfi-fss" {>= "0.0.2"}
+  "satyrographos" {>= "0.0.2"}
+]
+install: [
+  ["satyrographos" "opam" "install"
+   "-name" "class-jlreq"
+   "-prefix" "%{prefix}%"
+   "-script" "%{build}%/Satyristes"]
+]
```
Comment (na4zagin3) [inline packages/satysfi-class-jlreq/satysfi-class-jlreq.0.0.2/opam:20]: `url` section must be here
```suggestion
]
url {
  src:
    "https://github.com/abenori/satysfi-class-jlreq/archive/0.0.2.tar.gz"
  checksum: [
    "md5=5b9b9504df1e73a3924f5192e4b0fe96"
    "sha512=700aaae58aa0820a5e21637b570356cfafd78222d009e9ba2630b4db76c4b9fab40af99a97b5c6977a350d59e8ac8505f131bd5628b8f8010adad5f081708aeb"
  ]
}
```

**Comment by na4zagin3** [comment]
@abenori Thank you for sending the package!  Can you please check <https://github.com/na4zagin3/satyrographos-repo/issues/334> add add your comment there?  I'd like to apply `CC0-1.0+` licence to this repository.  Thanks!

**Comment by abenori** [comment]
Thank you. I replied. Sorry for my late reply.

**Comment by na4zagin3** [comment]
No worries!  Thank you for your confirmation!

**Pull request merged**